.github/workflows/python-ec2-default-test.yml

```
@@ -252,9 +257,26 @@ jobs:
           --instance-id ${{ env.MAIN_SERVICE_INSTANCE_ID }}
           --rollup'
 
+      - name: Validate CWAgent metrics
+        id: cwagent-metric-validation
+        if: (success() || (env.CUSTOM_METRICS_ENABLED == 'true' && steps.log-validation.outcome == 'failure')) && !cancelled()
```

I'm pretty sure this condition is wrong, it should be the following (I think)
```suggestion
        if: env.CUSTOM_METRICS_ENABLED == 'true' && (success() || steps.log-validation.outcome == 'failure') && !cancelled()
```

---

It will be removed along with the env. variable in the next push.